public class Exception2 {
  public static void main(String[] args) {
    try {
      System.out.println("nada");
    } catch(java.security.acl.LastOwnerException e) {
    }
  }
}